src/cmap/test/event.rs

```diff
@@ -64,8 +81,36 @@ impl CmapEventHandler for EventHandler {
     }
 }
 
+pub struct EventSubscriber<'a> {
+    _handler: &'a EventHandler,
```

Comment:
Could you add a doc comment explaining why `_handler` is here? My assumption is that it's to catch bugs where a subscriber outlives the handler at compile time, but I'm not positive

Reply:
Yep, done.